package.json

```diff
@@ -52,9 +52,9 @@
     "description": "Hackolade plugin for Google BigQuery Standard SQL",
     "disabled": false,
     "dependencies": {
-        "@google-cloud/bigquery": "5.7.1",
+        "@google-cloud/bigquery": "8.0.0",
         "@hackolade/sql-select-statement-parser": "0.0.20",
-        "lodash": "4.17.21"
+        "fs-extra": "11.3.0"
```

Comment:
I'm wondering why we can't just use Node.js's built-in fs API for this. What's the reasoning behind using fs-extra instead?